README.md

```diff
@@ -14,25 +14,37 @@ This repository contains Docker configuration aimed at Moodle developers and tes
 
 ## Prerequisites
 * [Docker](https://docs.docker.com) and [Docker Compose](https://docs.docker.com/compose/) installed
-* 3.25GB of RAM (to [run Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-setup#prerequisites))
+* 3.25GB of RAM (if you choose [Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-setup#prerequisites) as db server)
 
-## Example usage
+## Quick start
 
 ```bash
-# Set up path to code and choose a db server (pgsql/mssql/oracle/mysql)
+# Set up path to code
 export MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
-export MOODLE_DOCKER_DB=mssql
+# Choose a db server
```

Comment:
Please, keep its options inline in the comment: it's useful to cut&paste one among them to quickly change the DB server.

TIA,
Matteo

Reply:
I basically agree with you. But then this list must be maintained. The options table below says "pgsql, mariadb, mysql, mssql, oracle", the comment which I removed said "pgsql/mssql/oracle/mysql".

Reply:
I agree but the other option is to add a proper check in the shell command: IMHO it's simplier to maintain the list just in the README. Guessing no new DB type will be supported at least in middle term.

Reply:
OK, I changed this and updated the PR